### ISSUE TITLE
Network timeout on resource

### ISSUE THREAD

**Issue by ashresh** [addyosmani/a11y#73]
I am getting network timeout on resource when I am trying to run a11y through command line.
What is the parameter I need to set to solve this???

**Error:**
C:\workspace\a11y>a11y http://localhost:4200
Command failed: C:\Users\dkuma172\AppData\Roaming\npm\node_modules\a11y\node_modules\phantomjs-prebuilt\lib\phantom\bin\phantomjs.exe C:\Users\dkuma172\AppData\
Roaming\npm\node_modules\a11y\audits.js {"delay":1,"url":"http://localhost:4200"
,"width":1024,"height":768} --ignore-ssl-errors=true --ssl-protocol=tlsv1 --local-to-remote-url-access=true
Error code:408 Network timeout on resource. for http://localhost:4200/vendor.bundle.js

**Comment by DanielRuf** [comment]
See https://github.com/addyosmani/a11y/blob/7ebffa7b3bea489caafbd86da4147bb21c98ea47/audits.js#L12

I think you can provide your own audits.js file.
